extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -2114,7 +2118,7 @@ describes.repeated('', {
           'formFields[name]': 'John Miller',
           'formFields[email]': 'j@hnmiller.com',
         };
-        expect(form.submit).to.have.been.called;
+        expect(form.submit).to.have.not.been.called;
```

Comment:
I am confused, I think form.submit *should* be called since it's a non-XHR get, and the `form.submit` call is necessary to submit the request and navigate the browser, right?

Reply:
Ah I missed that there was an event that was propagating that would eventually cause the page to submit, thanks for explaining!